nav_msgs/msg/OdometryWithAcceleration.msg

```diff
@@ -2,8 +2,19 @@
 # The pose in this message should be specified in the coordinate frame given by header.frame_id.
 # The velocity in this message should be specified in the coordinate frame given by the child_frame_id.
 # The acceleration in this message should be specified in the coordinate frame given by the child_frame_id.
+#
+# If the covariance of the measurement is known, it should be filled in (if all you know is the
+# variance of each measurement, e.g. from a datasheet, just put those along the diagonal).
+# A covariance matrix of all zeros will be interpreted as "covariance unknown", and to use the
+# data a covariance will have to be assumed or gotten from some other source.
+#
+# If you have no estimate for one of the data elements, please set _the respective diagonal_ element
+# of the associated covariance matrix to -1.
+# If you are interpreting this message, please check for a value of -1 in the _diagonal_ elements
+# of each covariance matrix, and disregard the associated estimates.
+
 Header header
 string child_frame_id
 geometry_msgs/PoseWithCovariance pose
-geometry_msgs/TwistWithCovariance velocity
-geometry_msgs/TwistWithCovariance acceleration
+geometry_msgs/TwistWithCovariance twist
```

Comment:
This is a regression backwards. Twist is the datatype not the meaning.  This came up in the review of the Odometry message but was not changed there to velocity to keep backwards compatibility even though `velocity` was more semantically accurate.

We also can't change a core message like this without a lot of planning and preparation.

Reply:
I agree that velocity would be better, however if you actually want to access the velocity, you'd have to access `msg.velocity.twist` and there is the covariance `msg.velocity.covariance`

As I totally agree with your upper statement that 
> cost of changing these messages is very high

but I'd also prefer to keep this `msg.twist.twist` and `msg.twist.covariance` to avoid `twist` and `velocity` alongside each other, because we cannot have `msg.velocity.velocity` anyway